lib/cli/etcher.js

```diff
@@ -18,61 +18,115 @@
 
 const _ = require('lodash');
 const Bluebird = require('bluebird');
+const inquirer = require('inquirer');
 const isElevated = Bluebird.promisify(require('is-elevated'));
 const visuals = require('resin-cli-visuals');
-const form = require('resin-cli-form');
 const drivelist = Bluebird.promisifyAll(require('drivelist'));
+const chalk = require('chalk');
+const imageStream = require('etcher-image-stream');
 const writer = require('./writer');
 const utils = require('./utils');
 const options = require('./cli');
+const DynamicList = require('./dynamic-list');
+const DriveScanner = require('./drive-scanner');
 const EXIT_CODES = require('../src/exit-codes');
 
-isElevated().then((elevated) => {
+inquirer.registerPrompt('dynamicList', DynamicList);
+
+const scanner = new DriveScanner(2000);
+
+const driveToChoice = (drive, minSizeToEnable) => {
```

Comment:
Yeah, we should find a way to have all the logic necessary to display the drive options in the drive selection widget, while still have the knowledge about the image in the main file.